plat/nvidia/tegra/platform.mk

```diff
@@ -40,5 +40,5 @@ include ${SOC_DIR}/platform_${TARGET_SOC}.mk
 # modify BUILD_PLAT to point to SoC specific build directory
 BUILD_PLAT	:=	${BUILD_BASE}/${PLAT}/${TARGET_SOC}/${BUILD_TYPE}
 
-# enable signed comparison checks
-TF_CFLAGS	+= -Wsign-compare
+# platform cflags (enable signed comparisons, disable stdlib)
+TF_CFLAGS	+= -Wsign-compare -nostdlib
```

Comment:
Isn't ``-ffreestanding`` enough?

Reply:
For some reason we had to enable -nostdlib option

Reply:
That is strange. We may have to take a look at this, maybe we need to use that option in all platforms, not just here.